0-ad-history-mod-package/simulation/templates/units/mace/champion_infantry_spearman.xml

```diff
@@ -10,6 +10,11 @@
     <Requirements>
       <Techs datatype="tokens">-phase_city unlock_champion_infantry</Techs>
     </Requirements>
+    <History>
+        Hypaspists (translating to shield-bearers) were a type of elite infantry under Philip II and his son Alexander. Hellenistic successor states later adopted the concept. They were traditionally 3,000 men (three chiliarchies) strong. As an evolution of Macedonian foot companions, these soldiers were hand-picked or promoted for their merits and bravery. Because of their high skill and loyalty, they acted as the king's bodyguards and carried out special delicate missions such as guarding treasury or ambushing the enemy.
```

Comment:
```suggestion
    Hypaspists (translating to shield-bearers) were a type of elite infantry under Philip II and his son Alexander the Great. Hellenistic successor states later adopted the concept. They were traditionally 3,000 men (three chiliarchies) strong. As an evolution of Macedonian foot companions, these soldiers were hand-picked or promoted for their merits and bravery. Because of their high skill and loyalty, they acted as the king's bodyguards and carried out special, delicate missions such as guarding the treasury or ambushing the enemy.
```